frontend/packages/console-app/src/components/quick-starts/catalog/QuickStartCatalogFilter.tsx

```diff
@@ -0,0 +1,112 @@
+import * as React from 'react';
+import { useTranslation } from 'react-i18next';
+import {
+  SearchInput,
+  Flex,
+  FlexItem,
+  Dropdown,
+  DropdownToggle,
+  DropdownItem,
+  Checkbox,
+  Badge,
+} from '@patternfly/react-core';
+import { CaretDownIcon } from '@patternfly/react-icons';
+import { QuickStartStatus } from '../utils/quick-start-types';
+
+import './QuickStartCatalogFilter.scss';
+
+export type QuickStartFilteredStatus = Record<QuickStartStatus, boolean>;
+
+type QuickStartCatalogFilterProps = {
+  onChange: (val: string, statusFilters: QuickStartFilteredStatus) => void;
+  filterValue: string;
+  quickStartsCount: number;
+  quickStartStatusCount: Record<QuickStartStatus, number>;
+};
+
+const QuickStartCatalogFilter: React.FC<QuickStartCatalogFilterProps> = ({
```

Comment:
You can probably re-use `FilterToolbar` component here.

Reply:
According to our discussion, we won't be going with `FilterToolbar` since it would be difficult to use it here, as it would require a lot of effort to align it with the designs and it does not support custom dropdown items.